tools/ion-hash.go

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/amzn/ion-go/ion"
+	ionhash "github.com/amzn/ion-hash-go"
```

Comment:
I assume this is necessary because the containing folder is "ion-hash-go" rather than "ionhash"?
Should we create an "ionhash" folder and put all the source code in there?

Reply:
ionhash is the package we are referencing from "github.com/amzn/ion-hash-go" library. I need to do this or otherwise golint gives me an import lint error